javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,64 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+getShortestWord(danishWords);
+
+function getShortestWord(words) {
+    let shortestWord = '';
+    for (let i = 0; i < words.length; i++) {
+        if (shortestWord.length == 0) {
+            shortestWord = words[i];
+            continue;
+        }
+        if (words[i].length < shortestWord.length) {
+            shortestWord = words[i];
+        }
+    }
+    console.log(shortestWord);
+}
+
+
+//Find and count the Danish letters
+const danishLetters = ['å', 'æ', 'ø'];
+const obj = {};
+
+function getDanishLetters(str) {
+    let sum = 0;
+    const splittedString = str.split('');
+    for (let i = 0; i < danishLetters.length; i++) {
+        let count = 0;
+        for (let j = 0; j < splittedString.length; j++) {
```

Comment:
Nested loops should be avoided if possible because it makes the code more difficult to read.

In this case it is not necessary to do it. If you just map over the splittedString and check if that letter is in danishLetters, you achieve the same thing.

Reply:
changed nested loop using map